### ISSUE TITLE
Uncaught ReferenceError: datetime is not defined (nav.js:41)

### ISSUE THREAD

**Issue by arnars** [dubert/react-kronos#21]
I'm getting the stated error when selecting a date. (Uncaught ReferenceError: datetime is not defined (nav.js:41))

My React-code for the component is:

```
<Kronos
    onChange={this.props.onChange}
    date={this.props.date}
    format="MM-DD-YYYY"
    placeholder={this.props.header.label}
/>
```

The reason why `date` is using `this.props.date` and not `state`, is that the state is held in the parent component. I have tried changing it to state and add a default state, but the pattern is the same.

It is only the first time a date is selected in the component.

It might be locally but I was hoping that you could point me in the right direction as to what could be wrong.

All the best,
Anders

**Comment by felixakiragreen** [comment]
Hmm, I'm investigating it, I'll keep you posted. Passing `date` in as a prop shouldn't make a difference, so I'm not sure why it would be causing that issue right now.

Is `nav.js` from react-kronos or from your project? Can you include that code?

**Comment by arnars** [comment]
`nav.js` is from react-kronos in the `lib` folder. I tried looking into it to find out where the error came from, but I'm not that skilled within ES6.

With the react debugging console, I can see an object named `datetime` in the Kronos `states`, so it's there, somewhere.

Thank you for giving it a run, this error.

**Comment by felixakiragreen** [comment]
Is `this.props.date` defined? It needs to start with a date. Try doing: 

``` js
<Kronos
    onChange={this.props.onChange}
    date={this.props.date || new Date()}
    format="MM-DD-YYYY"
    placeholder={this.props.header.label}
/>
```

and see if there is still an error.

**Comment by arnars** [comment]
Okay - So it works both with `moment()` and `new Date()` - this does seem to remove the error. However now the date field is shown with the current date and not the placeholder.

Is there a way around this?

**Comment by felixakiragreen** [comment]
I FOUND IT!

It was a teeny tiny bug I missed when converting the project from coffeescript to es6.

Best of all, you won't need to use the `|| new Date()`. Updating now.

**Comment by felixakiragreen** [comment]
Fixed in 1.2.2.

Let me know if it doesn't work.

**Comment by arnars** [comment]
Mr. Luke Dubert.
You rock! 

(works perfectly in 1.2.2)
